020-Connect/010-Postgres/001-connect.mdx

```diff
@@ -0,0 +1,215 @@
+| `DROP STATISTICS`        | N             | N              | Y                 |
+| `ALTER RULE`             | N             | N              | Y                 |
+| `CREATE RULE`            | N             | N              | Y                 |
+| `DROP RULE`              | N             | N              | Y                 |
+| `ALTER TYPE`             | N             | N              | Y                 |
+| `CREATE TYPE`            | N             | N              | Y                 |
+| `DROP TYPE`              | N             | N              | Y                 |
+| `ALTER DOMAIN`           | N             | N              | Y                 |
+| `CREATE DOMAIN`          | N             | N              | Y                 |
+| `DROP DOMAIN`            | N             | N              | Y                 |
+| `ALTER FUNCTION`         | N             | N              | Y                 |
+| `CREATE FUNCTION`        | N             | N              | Y                 |
+| `DROP FUNCTION`          | N             | N              | Y                 |
+| `EXPLAIN`                | N             | N              | Y                 |
+| `REINDEX`                | N             | N              | Y                 |
+| `ALTER OPERATOR CLASS`   | N             | N              | Y                 |
+| `CREATE OPERATOR CLASS`  | N             | N              | Y                 |
+| `DROP OPERATOR CLASS`    | N             | N              | Y                 |
+| `CREATE EXTENSION`       | N             | N              | Y                 |
+| `DROP EXTENSION`         | N             | N              | Y                 |
+| `ALTER EXTENSION`        | N             | N              | Y                 |
+| `CREATE TEMPORARY TABLE` | N             | N              | Y                 |
+
+## Limitations
+
+### Unsupported statements and functions
+
+The following statements and/or functions are not available:
+
+- `SELECT user`
+- `SHOW password_encryption`
+- `SET ROLE`
+- `RESET ROLE`
+- `SET SESSION AUTHORIZATION`
+- `ALTER TABLESPACE`
+- `CREATE TABLESPACE`
+- `DROP TABLESPACE`
+
+### Connection limits
+
+- SQL over HTTP to a shared cluster has the rate limits in place, for a dedicated cluster there are none.
+- Shared cluster - TODO, amount of available connections
+- Dedicated cluster - no limit to connections
+
+### `xata_` Column prefix
+
+Xata reserves the `xata_` column prefix for [special columns](docs/concepts/data-model#special-columns) and rejects any custom created columns with the prefix.
+
+### sslmode
+
+Xata does not allow plain text connections, this may have an effect on the client you are using and the default that is inferred by such.
+If you face an issue with an error message as shown below, you will need to explicitly force SSL, by adding the sslmode to `require` in the connection string or pass in a parameterized way depending on the tool you are using.
```

Comment:
```suggestion
If you face an issue with an error message like the one shown below, you will need to explicitly force SSL, by setting the sslmode to `require` in the connection string or passing this setting in via a parameter depending on the tool you are using.
```

Reply:
Might benefit from showing an example of a connection string with the param: `?sslmode=require` or however it is constructed.

Reply:
Added example via https://github.com/xataio/mdx-docs/pull/281/commits/cafbafcd3c7e2e0a8f8cb2c3f22ea8076806dba0

and rewording via https://github.com/xataio/mdx-docs/pull/281/commits/3a77ac892ae0941bbcb181ccb72e070a198dd024 (GH UI refused to let me commit the suggestion 🤷 )